packages/client/src/api.ts

```diff
@@ -18,15 +18,19 @@ const getResponseErrorMessage = async (response: Response): Promise<string> => {
   }
 };
 
-export const requestWithFetch = async <T>(...args: Parameters<typeof fetch>): Promise<T> => {
-  const response = await fetch(...args);
-  if (!response.ok) {
-    throw new LogtoError({
-      message: await getResponseErrorMessage(response),
-      response,
-    });
-  }
+export const createRequester = (fetchFunction: typeof fetch = fetch) => {
```

Comment:
Should we detect `isNode` here? Looks like user needs to provide a fetch function when use it on NodeJS.